IamCredentials/composer.json

```diff
@@ -24,8 +24,7 @@
     },
     "require": {
         "php": ">=7.4",
-        "google/gax": "^1.7.0",
-        "google/common-protos": "^1.3.1||^2.0||^3.0"
```

Comment:
I wonder why was common-protos added here in the first place!

Reply:
I believe it's because it _used_ to not be a dependency of `google/gax`, but it is now!